pandas/core/series.py

```diff
@@ -1596,6 +1596,12 @@ def dot(self, other):
         else:  # pragma: no cover
             raise TypeError('unsupported type: %s' % type(other))
 
+    def __matmul__(self, other):
```

Comment:
can u update the .dot doc string that ‘@‘ operator is supported in >=3.5

Reply:
can you do this

Reply:
You mean update the .dot doc string? I believe that is done